src/Dropdown.elm

```diff
@@ -540,13 +547,9 @@ view (Config config) model (State state) =
             ]
                 ++ config.containerAttributes
 
-        filter item =
-            String.contains (state.filterText |> String.toLower)
```

Comment:
I would avoid the code de-dup here as the Elm compiler can reduce the `(state.filterText |> String.toLower)` to its final value.